### ISSUE TITLE
TLS handshake error when connecting to Ubuntu 22.04 LTS

### ISSUE THREAD

**Issue by DanielDobos1985** [appleboy/ssh-action#170]
Hello @appleboy,

We have a test (TST) and a recently pulled up production (PRD) server with Ubuntu 20.04 and 22.04 respectively.
The CI/CD workflow for DEV with **ssh-action@master** works fine for ages with the following arguments:
      
```
with:
        # The credentials used to authenticate the development server
        # are stored as GitHub secrets.
        host: ${{ secrets.SERVER_IP_ADDR_DEV }}
        username: ${{ secrets.SERVER_USER_DEV }}
        password: ${{ secrets.SERVER_PASSWORD_DEV }}
        envs: TARGET_DIR
        script: |
           cd $TARGET_DIR
           ... (more commands)
```

However, the PRD workflow with the same "with" section (having the "_PRD" credentials) gives the following error:

> 2022/07/14 07:06:43 ssh: handshake failed: ssh: no common algorithm for host key; client offered: [ssh-rsa-cert-v01@openssh.com ssh-dss-cert-v01@openssh.com ecdsa-sha2-nistp256-cert-v01@openssh.com ecdsa-sha2-nistp384-cert-v01@openssh.com ecdsa-sha2-nistp521-cert-v01@openssh.com ssh-ed25519-cert-v01@openssh.com ecdsa-sha2-nistp256 ecdsa-sha2-nistp384 ecdsa-sha2-nistp521 ssh-rsa ssh-dss ssh-ed25519], server offered: [rsa-sha2-512 rsa-sha2-256]

We have tried the "use_insecure_cipher: true" argument as well, but could not make it work.
We also use your **scp-action@master** in both the DEV and PRD CI/CD and it works fine.

Could you help us, please?
Best regards,
Daniel

**Comment by DanielDobos1985** [comment]
As mentioned in the following pull request, changing the version of the underlying **drone-ssh** solved the problem.
https://github.com/appleboy/ssh-action/pull/171

**Comment by oztechmuse** [comment]
Given the repo hasn't changed for so long, forking and hosting it yourself is an easy enough solution @DanielDobos1985 - but thanks for the solution - your fix worked for me on 22.04. In terms of the Docker tags - you could use appleboy/drone-ssh:latest or appleboy/drone-ssh:linux-amd64

**Comment by appleboy** [comment]
bump to v0.1.5 https://github.com/appleboy/ssh-action/releases/tag/v0.1.5
